packages/wallets/blocto/src/wallet.ts

```diff
@@ -4,6 +4,7 @@ import { BloctoWalletAdapter, BloctoWalletAdapterConfig } from './adapter';
 export const BloctoWalletName = 'Blocto' as WalletName;
 
 export const getBloctoWallet = (config: BloctoWalletAdapterConfig = {}): Wallet => ({
+    detectability: 'runtime-only',
```

Comment:
Depending on the runtime (eg. a desktop browser vs. a mobile webview) we can return a _different_ detectability. For instance, Blocto [says](https://docs.blocto.app/blocto-app/web3.js-integration-solana):

> Blocto injects web3 provider to web context as window.solana starting from version 2.9.0 on Android/iOS. To detect Blocto specifically, there is a isBlocto flag.

We could return `'installable'` on iOS and Android in this case.

Reply:
Returning a platform-specific value is what I was thinking as well. Could implement it as a getter, unless there's a reason to do it async? The `url` field could also be platform-specific, along with the general behavior of `connect`. For example, a dapp running in a mobile browser could launch a link to a mobile wallet instead of using an extension, possibly using the same adapter.

Reply:
I was thinking even something as simple as…

```
export const getBloctoWallet = (config: BloctoWalletAdapterConfig = {}): Wallet => ({
    detectability: somePlatformCheck() ? 'runtime-only' : 'installable',
    /* ... */
});
```